notebook/tests/selenium/utils.py

```diff
@@ -382,3 +394,71 @@ def trigger_keystrokes(browser, *keys):
             ac.perform()
         else:              # single key stroke. Check if modifier eg. "up"
             browser.send_keys(getattr(Keys, keys[0].upper(), keys[0]))
+
+def validate_notebook_state(notebook, mode, index):
```

Comment:
Can we have a docstring here? What exactly does the function validate? Maybe it can have a clearer name, for that matter?

Also, I think the whole thing could be simplified and cleaned up to Python code style standards. It doesn't look like it's ever called with `index=None`, so maybe it doesn't need to handle that.

Reply:
Worked on this on 3a0cb7b, I'm new to python so I couldn't find how to simplify the code by python standards so if there are some more specific changes I can work on them.